From e4f153b6f6316619d940ea5cfb18b33030d00e43 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Adri=C3=A1n=20Lavi=C3=B3s=20Gomis?= <adrian@lavios.eu>
Date: Sat, 13 Jan 2018 09:17:34 +0100
Subject: [PATCH 1/5] Revert "Load the toml file from the current directory by
 default"

This reverts commit e9faf4368ca0fe92fb6f4289943da20ce83fe18b.
---
 dnscrypt-proxy/config.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dnscrypt-proxy/config.go b/dnscrypt-proxy/config.go
index ca9031c..d4230d3 100644
--- a/dnscrypt-proxy/config.go
+++ b/dnscrypt-proxy/config.go
@@ -70,7 +70,7 @@ type BlockNameConfig struct {
 }
 
 func ConfigLoad(proxy *Proxy, config_file string) error {
-	configFile := flag.String("config", "dnscrypt-proxy.toml", "path to the configuration file")
+	configFile := flag.String("config", "/etc/dnscrypt-proxy/dnscrypt-proxy.toml", "path to the configuration file")
 	flag.Parse()
 	config := newConfig()
 	if _, err := toml.DecodeFile(*configFile, &config); err != nil {
-- 
2.15.1

